spec/spec_helper.rb

```diff
@@ -1,6 +1,14 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
-is_ci = (ENV.fetch('CI', nil) == 'true')
+
+module SpecHelper
+  class << self
+    def is_ci? = ENV.fetch('CI', nil) == 'true'
+
+    def use_headful_chrome? = ENV.fetch('HEADFUL_CHROME', nil).present?
+  end
+end
```

Comment:
Putting these as methods in a module (rather than local variables) makes it possible to access them from other files.

Reply:
I think that this is the first time that I have used Ruby's relatively newly added one-line endless method definition style.